storio-sample-app/src/main/java/com/pushtorefresh/storio/sample/db/resolvers/PersonGetResolver.java

```diff
@@ -0,0 +1,39 @@
+package com.pushtorefresh.storio.sample.db.resolvers;
+
+import android.database.Cursor;
+import android.support.annotation.NonNull;
+
+import com.pushtorefresh.storio.sample.db.entities.Car;
+import com.pushtorefresh.storio.sample.db.entities.Person;
+import com.pushtorefresh.storio.sample.db.tables.CarsTable;
+import com.pushtorefresh.storio.sample.db.tables.UsersTable;
+import com.pushtorefresh.storio.sqlite.operations.get.DefaultGetResolver;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class PersonGetResolver extends DefaultGetResolver<Person> {
+
+    // We expect that cursor will contain both Tweet and User: SQL JOIN
+    @NonNull
+    @Override
+    public Person mapFromCursor(@NonNull Cursor cursor) {
+        final Person person = Person.newPerson(
+                cursor.getLong(cursor.getColumnIndexOrThrow(UsersTable.COLUMN_ID)),
+                cursor.getString(cursor.getColumnIndexOrThrow(UsersTable.COLUMN_NICK))
+        );
+
+        List<Car> cars = new ArrayList<Car>();
+        while (cursor.moveToNext()) {
```

Comment:
Okay, so I'd suggest you to write `PersonGetResolver` in such manner as if you would work with it just for querying `person` table, so you would pass each query directly to the db and then just iterate over each `person` and retrieve his `cars`.

So you will be able to write any possible queries to the `persons` table and it'll work!

Try to change code to this and then I'll suggest ways of improving it :)